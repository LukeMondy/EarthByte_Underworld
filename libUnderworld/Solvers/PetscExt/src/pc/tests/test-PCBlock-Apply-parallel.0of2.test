EXEC 		= ./test-PCBlock-Apply-parallel.app
NPROCS 		= 2
TEST_TYPE       = tolerance_diff_pl,1.0e-10
