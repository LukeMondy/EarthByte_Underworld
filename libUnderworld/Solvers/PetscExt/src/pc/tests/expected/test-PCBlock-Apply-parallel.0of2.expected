test_pc_block_apply_USymTrans 
Process [0]
-1007.34
-112767
-62399.7
-45610.6
-37216.1
Process [1]
-32179.3
-28821.5
-26423.1
-24624.3
-23225.2
Process [0]
88
10.6667
6.11765
4.48
Process [1]
3.63636
3.12195
2.77551
2.52632
